redash/tasks/queries.py

```diff
@@ -277,14 +280,46 @@ def get_table_sample_data(existing_columns, data_source_id, table_name, table_id
         column_examples
     )
     models.db.session.commit()
+    logger.info(u"task=get_table_sample_data state=finished table_name=%s runtime=%.2f",
+                table_name, time.time() - start_time)
+
 
+@celery.task(name="redash.tasks.get_all_table_samples")
+def get_all_table_samples(data_source_id):
```

Comment:
Two minor things:

1. Let's call this task `refresh_samples` analog to `refresh_schemas`
2. Add a docstring describing what this does